cmd/openshift-install/agent/waitfor.go

```diff
@@ -71,8 +71,13 @@ func newWaitForBootstrapCompleteCmd() *cobra.Command {
 				logrus.Fatal(err)
 			}
 
+			agentAuthToken, err := agentpkg.FindAuthTokenFromAssetStore(assetDir)
```

Comment:
not a big blocking point, but I don't think it's necessary to re-introduce the duplication for retrieving the token: still, within the `NewCluster` method, the algo for retrieving the token could try look first in the asset store, and then in the cluster (if kubeconfig is not empty). It could probably help for simplifying a little bit the code.

Reply:
done